platform/source/minimal-printf/README.md

```diff
@@ -51,7 +51,7 @@ Minimal printf is configured by the following parameters defined in `platform/mb
 }
 ```
 
-By default, 64 bit integers, floating point  and FILE stream printing are enabled.
+By default, 64 bit integers and FILE stream printing are enabled.
```

Comment:
We should remove all references to FILE stream printing since the option to disable/enable has been removed.
This also applies to the tables at the end of the file

Reply:
I will have to re-build the example project and get new numbers for the six rows that do not include file stream.